### PR TITLE
Updated cache constraint

### PR DESCRIPTION
**Pull request by pkruithof**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -16,13 +16,13 @@
...
-    "guzzlehttp/guzzle": "~6.0",
-    "treehouselabs/cache": "~1.0"
+    "psr/log": "^1.0",
+    "psr/http-message": "^1.0",
+    "guzzlehttp/guzzle": "^6.0",
+    "treehouselabs/cache": "^2.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "~5.0"
+    "phpunit/phpunit": "^5.0"
```
Comment (fieg) [inline composer.json]: Projecten met een iets oudere phpunit versie (<5.0) kunnen deze library nu niet meer gebruiken?

Comment (pkruithof) [inline composer.json]: It's require-dev, only relevant when developing for this library.

Comment (fieg) [inline composer.json]: Ah yes, of course. And for the treehouselabs/cache constraint? Older versions really don't work anymore with <2.0 ?

Comment (pkruithof) [inline composer.json]: v3.2 just has the newer version required, nothing more, nothing less. It's not breaking for this library.

Comment (fieg) [inline composer.json]: Sure, but I'm talking about upgrade paths. If anyone wants to use this newer version, then they also have to update the cache library, which makes it harder to upgrade. If this library also works with lower versions of the cache bundle, then we should support that.

Comment (pkruithof) [inline composer.json]: The cache library is a dependency for this project, and will be upgraded automatically.

**Pull request merged**